packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -281,53 +281,96 @@ export class LitNodeClientNodeJs {
 
   // ==================== SESSIONS ====================
   /**
-   *
    * Try to get the session key in the local storage,
    * if not, generates one.
-   * @param { string } supposedSessionKey
-   * @return { }
+   *
+   * @param {string} [serializedSessionKeyPair] - Serialized session key pair
+   * @return {SessionKeyPair} - Session key pair
    */
-  getSessionKey = (supposedSessionKey?: string): SessionKeyPair => {
-    let sessionKey: any = supposedSessionKey ?? '';
-
-    const storageKey = LOCAL_STORAGE_KEYS.SESSION_KEY;
-    const storedSessionKeyOrError = getStorageItem(storageKey);
-
-    if (sessionKey === '') {
-      // check if we already have a session key + signature for this chain
-      // let storedSessionKey;
-      let storedSessionKey: any;
-
-      // -- (TRY) to get it in the local storage
-      if (storedSessionKeyOrError.type === 'ERROR') {
-        console.warn(
-          `Storage key "${storageKey}" is missing. Not a problem. Contiune...`
+  getSessionKey = (serializedSessionKeyPair?: string): SessionKeyPair => {
```

Comment:
does it need to be serialized? is it more convenient to have clients pass in the raw keypair object instead? it's not like the keypair is something that needs to travel over the wire i would presume.....(and hence would need serialization if it does travel over the wire)

Reply:
@hwrdtm 

since `getSessionKey` is currently called in more client-facing methods like `getSessionSigs`, the key pair doesn't have to be serialized. i'm open to enforcing the type in `getSessionKey`, `getSessionSigs`, and `signSessionKey` but this would be a breaking change.